File: fizzbuzz.cpp
#include <iostream>
using namespace std;

int main()
{
    int i;

    i = 0;
    while (++i <= 100)
    {
        if (!(i % 3))
            cout << "Fizz";
        if (!(i % 5))
            cout << "Buzz";
        if (i % 3 && i % 5)
            cout << i;
        cout << "\n";
    }
}

File: fizzbuzz.php
<?php 
$x = 1;

while ($x <= 100)
{
    if ($x % 3 == 0)
        echo "Fizz";
    if ($x % 5 == 0)
        echo "Buzz";
    if ($x % 3 && $x % 5)
        echo $x;
    echo "\n";
    $x++;
}
?>
